.github/workflows/cmake.yml

```diff
@@ -25,24 +25,41 @@ jobs:
           - {
               name: "Ubuntu Latest GCC",
               os: ubuntu-latest,
-              build_type: "Release", cc: "gcc", cxx: "g++"
+              build_type: "Release", cc: "gcc", cxx: "g++",
+              mpi_type: "mpich"
             }
           - {
               name: "Ubuntu Latest Clang",
               os: ubuntu-latest,
-              build_type: "Release", cc: "clang", cxx: "clang++"
+              build_type: "Release", cc: "clang", cxx: "clang++",
+              mpi_type: "mpich"
             }
           - {
               name: "macOS Latest Clang",
               os: macos-latest,
-              build_type: "Release", cc: "clang", cxx: "clang++"
+              build_type: "Release", cc: "clang", cxx: "clang++",
+              mpi_type: "mpich"
             }
     steps:
     - name: Checkout repository and submodules
       uses: actions/checkout@v2
       with:
         submodules: recursive
 
+    - name: Install Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.13'
+
+    - name: Install MPI
+      uses: mpi4py/setup-mpi@v1.3.2
+      with:
+        mpi: ${{matrix.config.mpi_type}}
```

Comment:
Tests are currently failing, I think we need `mpich-devel` or something as well

Reply:
[7012aa9](https://github.com/LLNL/Adiak/pull/38/commits/7012aa955807d2dd0255bcfedb1a2bb470e5a39c) Tried to fix, but I think workflow won't run automatically because it is a fork

Reply:
Looks like there's a syntax error in the file. It says
```
(Line: 67, Col: 11): Unexpected symbol: '"none"'. Located at position 22 within expression: matrix.config.mpi != "none"
```